### PR TITLE
fix: cache performance improvement

### PR DESCRIPTION
**Pull request by parthasl**
## Context

Cache performance is slower due to zip.

## Objective

Improve cache performance using tar + zstd.

## References

https://github.com/screwdriver-cd/launcher/pull/393

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by parthasl** [comment]
Later need to refactor to use tar + gozstd library.

File: screwdriver.yaml [hunk 1]
```diff
@@ -11,6 +11,11 @@ jobs:
         steps:
             - vet: go vet ./...
             - gofmt: ret=$(find . -name '*.go' | xargs gofmt -d) && echo "$ret" && test -z "$ret"
+            - pretest-setup: |
+                  wget -O hab.tar.gz 'https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-0.79.1-20190410220617-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux'
+                  tar -C . -ozxvf hab.tar.gz
+                  mv hab-*/hab /hab/bin/hab
+                  /hab/bin/hab pkg install -bf core/zstd
```
Comment (jithine) [inline screwdriver.yaml]: won't we have this in launcher already ?

Comment (parthasl) [reply screwdriver.yaml]: this is for pr tests in store-cli, will remove once launcher changes are merged

**Comment by parthasl** [comment]
looks like hab zstd is not available for mac.

**Review by vnugopal — DISMISSED** [review]

File: screwdriver.yaml [hunk 2]
```diff
@@ -11,6 +11,11 @@ jobs:
         steps:
             - vet: go vet ./...
             - gofmt: ret=$(find . -name '*.go' | xargs gofmt -d) && echo "$ret" && test -z "$ret"
+            - pretest-setup: |
+                  wget -q -O zstd-cli-linux.tar.gz 'https://bintray.com/screwdrivercd/screwdrivercd/download_file?file_path=zstd-cli-1.4.8-linux.tar.gz'
```
Comment (jithine) [inline screwdriver.yaml]: once the launcher PR get's merged, won't you have this binary already in place from launcher ? https://github.com/screwdriver-cd/launcher/pull/393/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R98

Comment (jithine) [reply screwdriver.yaml]: oh just saw your previous comment, so this will change prior to merge ?

Comment (parthasl) [reply screwdriver.yaml]: removed pretest-setup step

**Review by vnugopal — DISMISSED** [review]

**Review by jithine — APPROVED** [review]

**Pull request merged**